scan/templates/home/index.html

```diff
@@ -121,12 +127,62 @@ <h5 class="card-header-title">Pending Transactions</h5>
       </div>
     </div>
   </div>
+ </div>
+    <!--<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/chart.js@3.7.0/dist/chart.min.js"></script>
+<script>
+var ctxL = document.getElementById("myChart").getContext('2d');
+var myChart = new Chart(ctxL, {
+type: 'line',
+data: {
+labels: ["24 hours ago", "23 hours ago", "22 hours ago", "21 hours ago", "20 hours ago", "19 hours ago", "18 hours ago", "17 hours ago", "16 hours ago", "15 hours ago", "14 hours ago", "13 hours ago", "12 hours ago", "11 hours ago", "10 hours ago", "9 hours ago", "8 hours ago", "7 hours ago", "6 hours ago", "5 hours ago", "4 hours ago", "3 hours ago", "2 hours ago", "1 hour ago"],
+datasets: [
+{
+label: "STP",
+data: [28000, 48000, 40000, 190000, 60000, 270000, 90000, 48000, 40000, 19000, 86000, 270000, 90000, 48000, 40000, 190000, 86000, 2700, 90000, 48000, 40000, 19000, 86000, 27000],
+pointRadius: 2,
+pointHoverRadius: 4,
+tension: 0.4,
+backgroundColor: ['#007bff'],
+borderColor: ['#007bff'],
+borderWidth: 1,
+borderRadius: 3,
+},
+{
+label: "TP",
+data: [28, 48, 40, 19, 60, 27, 90, 48, 40, 19, 86, 27, 90, 48, 40, 19, 86, 27, 90, 48, 40, 19, 86, 27],
+pointRadius: 2,
+pointHoverRadius: 4,
+tension: 0.4,
+fill: true,
+backgroundColor: [
+'#22366B59',
+],
+borderColor: [
+'#22366B',
+],
+borderWidth: 1,
+borderRadius: 3,
+}
+]
+},
+options: {
+    scales: {y: {type: 'logarithmic',}, X: {ticks: {display: false},}},
+    animation: {duration: 5000,},
+    hover: { mode: 'index', intersect: false },
+    plugins: {
+    legend : false,
+    responsive: true,
+    tooltip: { mode: 'index', intersect: false },
+    title: {display:true, text:'TRANSACTIONS PER-HOUR', font:{family: "'Montserrat', sans-serif", weight: 500,}},
+             }
+         }
+});
+</script>-->
+
+<script>
+  setInterval(function () {
+    $('#update_Latest').load(location.href + ' #update_Latest');
+  }, 60000); //update every X seconds
```

Comment:
miliseconds I guess

Reply:
yes...  ill update all of these on my end..